pyproject.toml

```diff
@@ -20,15 +20,15 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
 dependencies = [
-  "pydantic~=1.10.8",
-  "jsonschema==3.2.0",
+  "pydantic~=1.10",
+  "jsonschema~=3.0",
   "PyYAML~=6.0",
-  "more-itertools==8.10.0",
-  "Jinja2==3.1.2",
-  "click>=7.1.2",
-  "python-dateutil==2.8.2",
-  "importlib_metadata==6.6.0",
-  "typing-extensions~=4.6.1",
+  "more-itertools~=8.0",
+  "Jinja2~=3.0",
```

Comment:
Are we certain this is not _too_ flexible? I think we are probably ok here since this will always be brought into dbt and Metricflow packages which will have their own restrictions, but I'm wondering if Jinja 3.2 could possibly include a breaking change for us

Reply:
It's a little hard to verify all package version combinations, so I figure going by tests in dbt / MF is reasonably sufficient.

Reply:
Also, if 3.2 includes a breaking change, it wouldn't be considered compatible with 3.0?